src/api.jl

```diff
@@ -773,12 +781,14 @@ function load_json(ctx::Context, database::AbstractString, engine::AbstractStrin
 end
 
 function load_model(ctx::Context, database::AbstractString, engine::AbstractString, models::Dict; kw...)
-    tx = Transaction(ctx.region, database, engine, "OPEN"; readonly = false)
-    actions = [_make_load_model_action(name, model) for (name, model) in models]
-    return _post(ctx, PATH_TRANSACTION; query = query(tx), body = body(tx, actions...), kw...)
+    queries = ["""def insert:rel:catalog:model["$(name)"] = \"\"\"$(models[name])\"\"\"""" for name in keys(models)]
```

Comment:
Same here, it's very important to use `$(repr(models[name]))` instead of `"""$(models[name])"""` to avoid (even accidental) injection attacks.

Otherwise, if the users' model file includes `"""` anywhere, this will be a syntax error. Does that make sense?

Reply:
(But, like above, you'll want to use https://github.com/RelationalAI/rai-sdk-issues/issues/68 to also address `%` in the user's model.)

Reply:
yep totally makes sense, thank you for pointing this out :pray: